manifest_test.go

```diff
@@ -25,7 +25,7 @@ func TestReadManifest(t *testing.T) {
 		t.Fatalf("Should have read Manifest correctly, but got err %q", err)
 	}
 
-	c, _ := gps.NewSemverConstraint(">=0.12.0, <1.0.0")
+	c, _ := gps.NewSemverConstraint("^0.12.0")
```

Comment:
I haven't absorbed the full context here yet, but just noting that these two aren't equivalent. By the definitions used in Masterminds/semver, `^0.12.0` is equivalent to `>=0.12.0, <0.13.0`.